__tests__/index.tests.js

```diff
@@ -39,3 +40,63 @@ describe('error 404', ()=>{
         })
     })
    
+    describe('GET /api', () => {
+      test('GET:200 sends an object with the endpoints to the client', () => {
+        return request(app)
+          .get('/api')
+          .expect(200)
+          .then((response) => {
+            expect(response.body.endpoints).toEqual(expect.any(Object));
+            expect(Object.keys(response.body.endpoints)).toEqual(
+              expect.arrayContaining(["GET /api", "GET /api/topics","GET /api/articles",  ]))
+              
+                     
+          });
+      });
+
+    }
+    )
+    describe('error 404 on GET /api bad path', ()=>{
+      test('GET:404', ()=>{
+          
+              return request(app)
+              .get('/apii')
+              .expect(404)
+     
+          })
+      })
+      describe('GET /api/articles/:article_id', () => {
```

Comment:
It looks like you have started the next task here as well. Please keep each task on its own branch. This is task 3 you submitted but the code here is task 4.